PencilCase/PencilCase/scripts/Pages/ProductEditor.ts

```diff
@@ -29,6 +29,15 @@ export class ProductEditor extends PageBase {
         super();
         this.title = ko.observable("Product Editor");
         this.pageId = Consts.Pages.ProductEditor.Id;
+        this.back = () => {
```

Comment:
For product editor, back is not just go back to previous page.
1. If is inline editing mode: a. If adding a new product, go back to the Product Management page. b. If editing one existed product, back to details mode.
2. If in details mode, just go back to the Product Management page.